src/pages/LoginPage/LoginPage.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+const LoginPage = () => {
```

Comment:
going to assume this `LoginPage` was not part of this PR otherwise there are many comments to be left here